### PR TITLE
Create Maintenance.html

### PR DESCRIPTION
**Pull request by jamblock**
Maintenace page has been added check it out

### CODE REVIEW

File: ErrorPages/Maintenance.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Mantenimiento</title>
+  </head>
+  <body>
+    <div class="col-md-12">
+      <h1>esta pagina esta ahorita en Mantenimiento</h1>
+      <<h2>el Mantenimiento puede durar algo</h2>
```
Comment (brucewayneims) [inline ErrorPages/Maintenance.html]: why there are "<" symbols ? could you remove them

File: ErrorPages/Maintenance.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <h3>asi que comanse algo mientras tanto</h3>
+    </div>
+  </body>
+  <script type="text/javascript">
+      function AlertaqueestaAqui() {
```
Comment (brucewayneims) [inline ErrorPages/Maintenance.html:15]: this function is not called by anything

**Review by brucewayneims — CHANGES_REQUESTED** [review]
could you check my comments lines 10, 15

File: ErrorPages/Maintenance.html (hunk 2 above)
Comment (jamblock) [reply ErrorPages/Maintenance.html:15]: now it is look at line 23 and the timeout

**Review by brucewayneims — APPROVED** [review]
all good then

**Pull request merged**